hail/python/hailctl/dataproc/modify.py

```diff
@@ -54,48 +53,43 @@ def main(args, pass_through_args):
             print("Updating cluster '{}'...".format(args.name))
             check_call(cmd)
 
-    if (args.jar is not None):
-        print('gcloud jar update command(s):')
-        _scp_and_sudo_move(args.jar, args.name, '/home/hail/hail.jar', args.zone)
-    if (args.zip is not None):
-        print('gcloud zip update command(s):')
-        _scp_and_sudo_move(args.zip, args.name, '/home/hail/hail.zip', args.zone)
-
-
-# user doesn't have access to /home/hail/ so we copy then use sudo
-def _scp_and_sudo_move(source, destination_host, destination, zone):
-    cmds = []
-    if source.startswith("gs://"):
-        cmds.append([
-            'gcloud',
-            'compute',
-            'ssh',
-            '{}-m'.format(destination_host),
-            '--zone={}'.format(zone),
-            '--',
-            'sudo gsutil cp {} {}'.format(source, destination)
-        ])
-    else:
-        cmds.extend([
-            [
-                'gcloud',
-                'compute',
-                'scp',
-                '--zone={}'.format(zone),
-                source,
-                '{}-m:/tmp/foo'.format(destination_host)
-            ],
-            [
+    if (args.wheel is not None):
+        wheel = args.wheel
+        cmds = []
+        if wheel.startswith("gs://"):
+            cmds.append([
                 'gcloud',
                 'compute',
                 'ssh',
-                '{}-m'.format(destination_host),
-                '--zone={}'.format(zone),
+                '{}-m'.format(args.name),
+                '--zone={}'.format(args.zone),
                 '--',
-                'sudo mv /tmp/foo {}'.format(destination)
-            ]
-        ])
+                f'sudo gsutil cp {wheel} /home/hail/ && '
+                'sudo /opt/conda/default/bin/pip uninstall -y hail && '
+                'sudo /opt/conda/default/bin/pip install --no-deps /home/hail/*.whl'
```

Comment:
I think we should at least be consistent about `--no-deps` versus `--no-dependencies` below.

Reply:
Somehow we're installing hail's dependencies, right? Can we replicate that logic in modify so that modify works as one would expect?